blueprints/app/files/tests/helpers/setup-application-test.js

```diff
@@ -0,0 +1,5 @@
+import { setupApplicationTest } from 'ember-qunit';
+
+export default function(hooks) {
```

Comment:
I thought you wanted to make it a named export? 🤔

Reply:
Yup, I'll change to 1 file (`test-setup-functions.js`) with 3 exports.